atm.js

```diff
@@ -1,21 +1,62 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
+$( document ).ready( function(){
+
+var cBal = 0
+var sBal = 0
+
+//Had issues making this work
+// $("div.zero").css({
+//       "background-color": "000000",
```

Comment:
it's because `000000` is not a valid color. use the `#` to indicate that this is a hex color - `#000000`